jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java

```diff
@@ -131,6 +142,16 @@ public void onTaskSelectionEvent(@Observes final TaskSelectionEvent event) {
         this.currentTaskId = event.getTaskId();
         this.currentServerTemplateId = event.getServerTemplateId();
         this.currentContainerId = event.getContainerId();
+        this.currentTaskSummary =
```

Comment:
the idea is the refresh have to do as much as possible. If the task is not persisted, the details would be the one that the task refresh event holds. But if the task is persisted the method getTask return the last task data. To allow the refresh operation in both cases, I thought to have the last load task details in the currenTaskSummary to be used when there's no task details  at DB. This task selection method will try to access to the last task data and call the setTaskDetails with the best data maintaining the previous behavior. Here we have to evaluate is we can rely always on the event data and avoid to call the getTask Method in any case. What do you think @cristianonicolai about this modification? does it make sense?